siva/library.go

```diff
@@ -61,23 +61,20 @@ type LibraryOptions struct {
 	// Performance enables performance options in read only git repositories
 	// (ExclusiveAccess and KeepDescriptors).
 	Performance bool
-	// DontGenerateID stops the library from generating a library ID if it's
-	// not provided or already in metadata.
-	DontGenerateID bool
+	// Metadata flags if the library must use stored metadata.
+	StoredMetadata bool
```

Comment:
Loading metadata should be the default behavior. I would enforce it for now and if needed we can add a `DisableMetadata` option when we find the use case.